tools/qupath_roi_splitter/qupath_roi_splitter.xml

```diff
@@ -3,10 +3,13 @@
     <requirements>
         <requirement type="package" version="3.0.1">geojson</requirement>
         <requirement type="package" version="1.24.2">numpy</requirement>
-        <requirement type="package" version="4.7.0">libopencv</requirement>
+        <requirement type="package" version="4.7.0">opencv</requirement>
         <requirement type="package" version="2.0.0">pandas</requirement>
     </requirements>
      <command detect_errors="exit_code"><![CDATA[
+        export LIBGL_PATH=$(find /usr -name libGL.so.1) &&
```

Comment:
Its running in a container, and this container probably has no libgl.
Can you create a container here please: https://github.com/BioContainers/multi-package-containers/

With this workarund we can create a libgl container.

Reply:
thanks!

Reply:
libGL isn't availiable as a conda package, so I assume this will not work?

there is a headless opencv package availiable, which should work for use cases like servers and docker containers. 
It's availiable via fastai (not cia conda-forge).

https://github.com/BioContainers/multi-package-containers/pull/2585